modules/standard/ChapelIO.chpl

```diff
@@ -853,6 +800,12 @@ module ChapelIO {
     }
   }
 
+  pragma "no doc"
+  proc chpl_stringify_wrapper(const args ...):string {
```

Comment:
Why create a wrapper instead of adding `use IO only stringify;` at module scope in `Error` and using `stringify()` directly?

Reply:
Because that led to problems with resolution of internal/standard modules (order of resolution issues).